### ISSUE TITLE
Created public link not visible after re-opening the share options

### ISSUE THREAD

**Issue by a-mlbz** [owncloud/core#34375]
Hey guys,

public links aren't displayed anymore when you re-open the share options in the GUI with Chrome 72 and OSX.
Same issue for Firefox 65, but I found a work-around - you need to press CTRL+R from time to time.
Browser cache und cookies were deleted before, but this didnt help.

### Steps to reproduce
1. Create Public Link Share
2. Close the share options
3. Open the shape options again
4. No Public Link Share displayed

### Expected behaviour
Public Link Share should be visible

### Actual behaviour
Public Link Share not visible

### Server configuration
**Operating system**: CentOS 7

**Web server:** Apache 2.4

**Database:** MySQL

**PHP version:** 7.0

**ownCloud version:** 10.0.10.4

**Updated from an older ownCloud or fresh install:** updated from 9.1.8

**Where did you install ownCloud from:** Archive from owncloud.org

**Signing status (ownCloud 9.0 and above):** ?

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results into https://gist.github.com/ and puth the link here.
```
"No errors have been found."

**The content of config/config.php:**

```
    "config": {
        "instanceid": "ocb6004c626d",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "ocqa2.domain.com",
            "ocasqa2.domain.com",
            "ocqa.domain.com"
        ],
        "logfile": "\/var\/log\/owncloud\/owncloud.log",
        "loglevel": 2,
        "datadirectory": "\/apps\/owncloud\/data",
        "dbtype": "mysql",
        "version": "10.0.10.4",
        "dbname": "owncloud",
        "dbhost": "ocdbqa2.domain.com",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "ldapIgnoreNamingRules": false,
        "theme": "",
        "maintenance": false,
        "forcessl": true,
        "logtimezone": "Europe\/Berlin",
        "default_language": "en_GB",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trashbin_retention_obligation": "auto",
        "mail_smtpmode": "php",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "memcache.distributed": "\\OC\\Memcache\\APCu",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "redis": {
            "host": "127.0.0.1",
            "port": 6379
        },
        "singleuser": false
```

**List of activated apps:**

```
Enabled:
  - activity: 2.4.1
  - comments: 0.3.0
  - configreport: 0.1.1
  - dav: 0.4.0
  - federatedfilesharing: 0.3.1
  - federation: 0.1.0
  - files: 1.5.1
  - files_external: 0.7.1
  - files_pdfviewer: 0.10.0
  - files_sharing: 0.11.0
  - files_texteditor: 2.3.0
  - files_trashbin: 0.9.1
  - files_versions: 1.3.0
  - files_videoplayer: 0.9.8
  - gallery: 16.1.1
  - market: 0.3.0
  - notifications: 0.3.5
  - provisioning_api: 0.5.0
  - systemtags: 0.3.0
  - templateeditor: 0.4.0
  - updatenotification: 0.2.1
  - user_ldap: 0.13.0
Disabled:
  - encryption
  - external
  - firstrunwizard
  - user_external
```

**Are you using external storage, if yes which one:** No

**Are you using encryption:** No

**Are you using an external user-backend, if yes which one:** LDAP

#### LDAP configuration (delete this part if not used)

```
+-------------------------------+--------------------------------------------------------------------------------------------------------------------------------------+
| Configuration                 | s01                                                                                                                                  |
+-------------------------------+--------------------------------------------------------------------------------------------------------------------------------------+
| hasMemberOfFilterSupport      | 1                                                                                                                                    |
| hasPagedResultSupport         |                                                                                                                                      |
| homeFolderNamingRule          |                                                                                                                                      |
| lastJpegPhotoLookup           | 0                                                                                                                                    |
| ldapAgentName                 | user                                                                                                                             |
| ldapAgentPassword             | ***                                                                                                                                  |
| ldapAttributesForGroupSearch  |                                                                                                                                      |
| ldapAttributesForUserSearch   |                                                                                                                                      |
| ldapBackupHost                |                                                                                                                                      |
| ldapBackupPort                |                                                                                                                                      |
| ldapBase                      | DC=domain,DC=com                                                                                                                        |
| ldapBaseGroups                | DC=domain,DC=com                                                                                                                        |
| ldapBaseUsers                 | DC=domain,DC=com                                                                                                                        |
| ldapCacheTTL                  | 600                                                                                                                                  |
| ldapConfigurationActive       | 1                                                                                                                                    |
| ldapDynamicGroupMemberURL     |                                                                                                                                      |
| ldapEmailAttribute            | mail                                                                                                                                 |
| ldapExperiencedAdmin          | 0                                                                                                                                    |
| ldapExpertUUIDGroupAttr       |                                                                                                                                      |
| ldapExpertUUIDUserAttr        |                                                                                                                                      |
| ldapExpertUsernameAttr        | sAMAccountName                                                                                                                       |
| ldapGroupDisplayName          | cn                                                                                                                                   |
| ldapGroupFilter               | (&(|(objectclass=group))(|(cn=....)))                                                                                           |
| ldapGroupFilterGroups         | ....                                                                                                                                 |
| ldapGroupFilterMode           | 1                                                                                                                                    |
| ldapGroupFilterObjectclass    | group                                                                                                                                |
| ldapGroupMemberAssocAttr      | member                                                                                                                               |
| ldapHost                      | ldap://dc.domain.com                                                                                                                    |
| ldapIgnoreNamingRules         |                                                                                                                                      |
| ldapLoginFilter               | (&(memberOf:1.2.840.113556.1.4.1941:=CN=....,OU=....,OU=....,OU=....,DC=domain,DC=com)(sAMAccountName=%uid))     |
| ldapLoginFilterAttributes     |                                                                                                                                      |
| ldapLoginFilterEmail          | 0                                                                                                                                    |
| ldapLoginFilterMode           | 1                                                                                                                                    |
| ldapLoginFilterUsername       | 0                                                                                                                                    |
| ldapNestedGroups              | 0                                                                                                                                    |
| ldapOverrideMainServer        | 0                                                                                                                                    |
| ldapPagingSize                | 500                                                                                                                                  |
| ldapPort                      | 389                                                                                                                                  |
| ldapQuotaAttribute            | 2 GB                                                                                                                                 |
| ldapQuotaDefault              |                                                                                                                                      |
| ldapTLS                       | 0                                                                                                                                    |
| ldapUserDisplayName           | displayname                                                                                                                          |
| ldapUserDisplayName2          |                                                                                                                                      |
| ldapUserFilter                | (&(|(objectclass=user))(|(|(memberof=CN=....,OU=....,OU=.....,OU=....,DC=domain,DC=com)(primaryGroupID=130246)))) |
| ldapUserFilterGroups          | ....                                                                                                                            |
| ldapUserFilterMode            | 0                                                                                                                                    |
| ldapUserFilterObjectclass     | user                                                                                                                                 |
| ldapUserName                  | samaccountname                                                                                                                       |
| ldapUuidGroupAttribute        | auto                                                                                                                                 |
| ldapUuidUserAttribute         | auto                                                                                                                                 |
| turnOffCertCheck              | 0                                                                                                                                    |
| useMemberOfToDetectMembership | 1                                                                                                                                    |
+-------------------------------+--------------------------------------------------------------------------------------------------------------------------------------+

```

### Client configuration
**Browser:** Chrome 72

**Operating system:** MacOS 10.14

#### Web server error log
```
No errors
```

#### ownCloud log (data/owncloud.log)
```
No errors
```

#### Browser log
```
No errors
```

**Comment by phil-davis** [comment]
I tried with Chrome 72 on Ubuntu 18.04 and the share options dialog opens/closes and keeps displaying the public link information.
@a-mlbz do you have another OS (Ubuntu, Windows...) that you can try from? And let us know if it is OSX-specific or...

**Comment by ownclouders** [comment]
[GitMate.io](https://gitmate.io) thinks possibly related issues are https://github.com/owncloud/core/issues/8559 (public links no longer work after restore of shared folder), https://github.com/owncloud/core/issues/13430 (Can't open pictures with shared link), https://github.com/owncloud/core/issues/16436 (The options/permissions in federated sharing are not visible), https://github.com/owncloud/core/issues/18251 (Create Links for Shared Files or Folders (not a public share)), and https://github.com/owncloud/core/issues/4062 (Public share icons not visible on Firefox).

**Comment by a-mlbz** [comment]
Hey phil,
To get the full picture:

We have a productive owncloud and QA environment.  Upgrade was first done on QA end of November and tested with Win and OSX using Firefox 64 and Chrome 71. No issues found.

Now I upgraded the PROD server from 9.1.8 to 10.0.10 last Saturday. After the upgrade we noticed the following issues:
- **Windows + Chrome 72**: Public Shares not visible when re-opening the Public Share options
- **OSX + Chrome 72**: Same as with Windows. Additionally file uploads are not possible ("Not enough free space, you are uploading X MB but only 0 B is left")
- **Windows + Firefox 65**: Public Shares not displayed, CTRL+R fixes it
- **OSX + Firefox 65**: Public Shares not displayed, CTRL+R fixes it

Then I rolled back and loaded the snapshots I created before the upgrade. 

So only QA is still on 10.0.10.

I just tested again using OSX and Chrome 72. Same issue. Installing Chrome 72 fixes the issue.
Same on Windows and Chrome 72. 
I noticed one thing. The icon of the directory which is shared, loses the shared symbol as soon as you try to manage the share. That doesnt happen in Firefox. See:

![share icon missing](https://user-images.githubusercontent.com/47319368/52222597-2dd21100-28a4-11e9-8b56-d5908ba1902e.JPG)

Any ideas?

Cheers
